adobe/implementation/toroid.hpp

```diff
@@ -227,7 +227,7 @@ class byte_toroid_t : boost::noncopyable {
 
     void purify(toroid_node_t* p) {
         toroid_header_t* c(column_of(p));
-        int x(p->color_m);
+        auto x(p->color_m);
```

Comment:
Surely not needed for C++20 support?  Why is it even an improvement?  

Aside: we really don't use copy-initialization anymore?  This syntax makes it look like a function declaration.

Reply:
This silences a warning because `x` is char type and later uses will narrow back to a char type.